docs/src/components/home/References.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+const References = () => {
+  return (
+    <Grid
+      container
+      spacing={2}
+      alignItems="center"
+      justifyContent="center"
+      sx={{
+        textAlign: 'center',
+        '& img': { maxWidth: '100%', px: 1, filter: 'opacity(0.475)' },
+      }}
+    >
+      <Grid item xs={4} md={2}>
+        <img src="/static/branding/pricing/coursera.svg" alt="Coursera logo" />
```

Comment:
Feel free to reorganize these assets. We could for instance use `/static/company/` for the assets related to the branding of our company.